cmd/neofs-cli/modules/dump.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"github.com/mr-tron/base58"
+	"github.com/nspcc-dev/neofs-node/pkg/services/control"
+	controlSvc "github.com/nspcc-dev/neofs-node/pkg/services/control/server"
+	"github.com/nspcc-dev/neofs-sdk-go/util/signature"
+	"github.com/spf13/cobra"
+)
+
+const (
+	dumpFilepathFlag     = "path"
+	dumpIgnoreErrorsFlag = "no-errors"
+)
+
+var dumpShardCmd = &cobra.Command{
+	Use:   "dump",
+	Short: "Dump objects from shard",
+	Long:  "Dump objects from shard to a file",
+	Run:   dumpShard,
+}
+
+func dumpShard(cmd *cobra.Command, _ []string) {
+	key, err := getKey()
+	exitOnErr(cmd, err)
+
+	body := new(control.DumpShardRequest_Body)
+
+	rawID, err := base58.Decode(shardID)
+	exitOnErr(cmd, errf("incorrect shard ID encoding: %w", err))
+	body.SetShardID(rawID)
+
+	p, _ := cmd.Flags().GetString(dumpFilepathFlag)
```

Comment:
should we validate the path? (this is _path_, this is not empty, etc)

same for `restore`

Reply:
This path should exist on storage node side, which can be remote node, thus path can't be checked.

Reply:
i just meant to check the basic rules: go lib must be able to parse it correctly; path is not `""`; etc

as i see, nothing would stop cli from sending RPC call to the node that asks to dump everything to `some/awesome\path_on-my machine.exe` path